canary/producer-cpp/kvs_log_configuration

```diff
@@ -0,0 +1,16 @@
+log4cplus.rootLogger=DEBUG, KvsConsoleAppender, KvsFileAppender
```

Comment:
This file is not necessary since we end up copying as part of CMakeLists.txt file. Remove the file.